src/StructuredLogger/BinaryLog.cs

```diff
@@ -165,7 +165,7 @@ public static Build ReadBuild(Stream stream, Progress progress, byte[] projectIm
 
             if (errorByType.Any(i => i != 0))
             {
-                string summary = string.Join(", ", errorByType.Where((count, index) => count > 0).Select((count, index) => $"{((ReaderErrorType)index)}: {count}"));
+                string summary = string.Join(", ", errorByType.Where((count, index) => count > 0).Select((count, index) => $"{((ReaderErrorType)index)}: {count} cases"));

```

Comment:
I'm finding the existing message confusing. For example, `Skipped some data unknown to this version of Viewer. 7 cases encountered (UnknownEventType: 7).` to me suggests that `7` is the event type. Making it clearer that the number is the count.

Reply:
Indeed very confusing. Thanks!